components/bulletins/BulletinCard.vue

```diff
@@ -50,55 +50,56 @@ export default {
                 bulletinCard: true,
             }
         },
+        showButton() {
+            return false
+        },
     },
 }
 </script>
 
 <style lang="postcss" scoped>
 .bulletinCard {
-    @apply relative flex flex-col items-center font-serif rounded-3xl border-4;
-    width: 47%;
-    @media (min-width: 415px) {
-        width: 214px;
-    }
-    border-color: #4b4b4b;
-    color: #4b4b4b;
+    @apply relative flex flex-col justify-center items-center rounded-xl;
+    min-width: 236px;
+    min-height: 310px;
+    color: #a9a6d9;
+    background: #1f1c3b;
 }
 
 .bulletinCard:hover {
-    border-color: #413f5a;
+    border-color: #746bb8;
+    background: #352d66;
+    color: #f0ebf5;
 }
 
 /* A trick to set height to be equal to width. */
 /* Checkout: https://stackoverflow.com/a/6615994/7969188 */
 .bulletinCard > .dummy {
     margin-top: 230%;
-    @media (min-width: 370px) {
+    @media (min-width: 375px) {
         margin-top: 170%;
     }
 }
 
 .bulletinCard__content {
     @apply absolute;
+    top: 120px;
+    padding: inherit;
 }
 
 .bulletinCard__content > h2 {
     @apply text-base text-center mt-6 mb-5 font-bold;
-    @media (min-width: 415px) {
+    font-family: 'Noto Serif TC';
+    @media (min-width: 375px) {
         @apply text-xl mt-8 mb-5;
     }
-    color: #808080;
-}
-
-.bulletinCard__content:hover > h2 {
-    color: #7568f6;
 }
 
 .bulletinCard__content > p {
     @apply mx-2;
     font-size: 0.6rem;
-    color: #c7c7c7;
-    @media (min-width: 415px) {
+    font-family: 'Source Sans Pro';
```

Comment:
**question (non-blocking):** it seems like this property has been set from the global scope. Are there other reasons for adding `font-family` here?

Reply:
Removed